docs/03 - Implementing the Anchor Server/Communication/Events Schema.yml

```diff
@@ -156,16 +167,18 @@ components:
             $ref: '#/components/schemas/StellarTransaction'
         external_transaction_id:
           type: string
+          description: "The id of an external transaction to be handled by the Anchor Backend."
         custodial_transaction_id:
           type: string
         source_account:
           type: string
-          description: The SEP-12 user ID of the source account.
+          description: "The Stellar account who sent the payment that originated this event. Should be present if this event was originated from a Stellar payment."
```

Comment:
This description could be confused with a transaction's source account, which could be different than the source of funds. Can we make it clear that this is the account whose balance was debited?

Reply:
Updated in https://github.com/stellar/java-stellar-anchor-sdk/pull/412/commits/df6d62803935be72225784b835eabd1797eacfab.